Microsoft.Toolkit.Uwp.UI.Media/Brushes/GeometrySurfaceBrush.cs

```diff
@@ -0,0 +1,383 @@
+        }
+
+        /// <summary>
+        /// Instance handler for the changes to the FillBrush dependency property.
+        /// </summary>
+        private void OnFillBrushChanged()
+        {
+            _fillBrushUpdateListener?.Detach();
+            _fillBrushUpdateListener = null;
+
+            if (FillBrush != null)
+            {
+                _fillBrushUpdateListener = new WeakEventListener<RenderCanvasBrushBase, object, EventArgs>(FillBrush)
+                {
+                    OnEventAction = async (instance, source, args) =>
+                    {
+                        await Window.Current.Dispatcher.RunAsync(CoreDispatcherPriority.Normal, () =>
+                        {
+                            OnSurfaceBrushUpdated();
+                        });
+                    }
+                };
+
+                FillBrush.Updated += _fillBrushUpdateListener.OnEvent;
+
+                OnSurfaceBrushUpdated();
+            }
+        }
+
+        /// <summary>
+        /// BackgroundBrush Dependency Property
+        /// </summary>
+        public static readonly DependencyProperty BackgroundBrushProperty = DependencyProperty.Register(
+            "BackgroundBrush",
+            typeof(RenderCanvasBrushBase),
+            typeof(GeometrySurfaceBrush),
+            new PropertyMetadata(null, OnBackgroundBrushChanged));
+
+        /// <summary>
+        /// Gets or sets the brush with which the background of the Geometry surface will be rendered.
+        /// </summary>
+        public RenderCanvasBrushBase BackgroundBrush
+        {
+            get => (RenderCanvasBrushBase)GetValue(BackgroundBrushProperty);
+            set => SetValue(BackgroundBrushProperty, value);
+        }
+
+        /// <summary>
+        /// Handles changes to the BackgroundBrush property.
+        /// </summary>
+        /// <param name="d"><see cref="GeometrySurfaceBrush" />.</param>
+        /// <param name="e">DependencyProperty changed event arguments.</param>
+        private static void OnBackgroundBrushChanged(DependencyObject d, DependencyPropertyChangedEventArgs e)
+        {
+            var geometrySurfaceBrush = (GeometrySurfaceBrush)d;
+            geometrySurfaceBrush.OnBackgroundBrushChanged();
+        }
+
+        /// <summary>
+        /// Instance handler for the changes to the BackgroundBrush dependency property.
+        /// </summary>
+        private void OnBackgroundBrushChanged()
+        {
+            _bgBrushUpdateListener?.Detach();
+            _bgBrushUpdateListener = null;
+
+            if (BackgroundBrush != null)
+            {
+                _bgBrushUpdateListener = new WeakEventListener<RenderCanvasBrushBase, object, EventArgs>(BackgroundBrush)
+                {
+                    OnEventAction = async (instance, source, args) =>
+                    {
+                        await Window.Current.Dispatcher.RunAsync(CoreDispatcherPriority.Normal, () =>
```

Comment:
Make sure to update all `Dispatcher` usage to `DispatcherQueue`

Reply:
Done.